doc/getstarted/build_and_install/docker_install_en.rst

```diff
@@ -104,3 +104,70 @@ container:
 
 Then we can direct our Web browser to the HTML version of source code
 at http://localhost:8088/paddle/
+
+
+Development Using Docker
+------------------------
+
+Develpers can work on PaddlePaddle using Docker.  This allows
+developers to work on different platforms -- Linux, Mac OS X, and
+Windows -- in a consistent way.
+
+The general development workflow with Docker and Bazel is as follows:
+
+1. Get the source code of Paddle:
+
+   .. code-block:: bash
+
+      git clone --recursive https://github.com/paddlepaddle/paddle
+
+
+2. Build a development Docker image `paddle:dev` from the source code.
+   This image contains all the development tools and dependencies of
+   PaddlePaddle.
+
+
+   .. code-block:: bash
+
+      cd paddle
+      docker build -t paddle:dev -f paddle/scripts/docker/Dockerfile .
+
+
+3. Run the image as a container and mounting local source code
+   directory into the container.  This allows us to change the code on
+   the host and build it within the container.
+
+   .. code-block:: bash
+
+      docker run \
+       -d # run the container in background mode \
```

Comment:
The backslash continues the comments to next lines, thus makes the ```docker run``` command wrong.